chore: update doc syncing instructions (#2043)
sveltejs/svelte.dev: README.md

```
pnpm install
cd apps/svelte.dev

> pnpm sync-docs -p # the -p flag clones our git repos. Otherwise, it expects symlinked local repositories in `apps/svelte.dev/repos`

Author: ```suggestion
pnpm sync-docs -p # use the -p flag to clone the git repos into `apps/svelte.dev/repos`
```
